compiler/ir.py

```diff
@@ -242,11 +243,11 @@ def compile_command_to_DFG(fileIdGen, command, options, redirections=None):
         command_invocation
     )
     if io_info is None:
-        raise Exception(
+        raise UnparallelizableError(
             f"InputOutputInformation for {format_arg_chars(command)} not provided so considered side-effectful."
         )
     if io_info.has_other_outputs():
-        raise Exception(
+        raise UnparallelizableError(
```

Comment:
Konstantinos (from old branch): Maybe also print what is the other output?

Reply:
I think I may need some help with determining what the other outputs to pass it along in the exception. I see that in `io_info.has_other_outputs()`, it is going through a for-loop checking if each entry is a "other output" [has_other_outputs() function (line 182)](https://github.com/binpash/annotations/blob/main/pash_annotations/annotation_generation/datatypes/InputOutputInfo.py); I'm not sure if there's already a function in the class that returns these "other outputs", but if not, would it be reasonable to change this function or add another function like this to return a list of those outputs in the class?

Reply:
I am not sure if there is such a function, but if not, you could certainly make a new one that returns a list of those outputs!